site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2291,9 +2312,9 @@ public function ajaxSaveGradedComponent($gradeable_id) {
         $logger_params = [
             "course_semester" => $this->core->getConfig()->getTerm(),
             "course_name" => $this->core->getDisplayedCourseName(),
-            "gradeable_id" => $gradeable_id,
-            "grader_id" => $this->core->getUser()->getId(),
-            "component_id" => $component_id,
+            "gradeable_id" => $gradeable->getId(),
```

Comment:
I think these should get reverted, as the gradeable_id and component_id are passed as parameters or POST variables to the function, and we should keep them as such.